### DISCUSSION TITLE
Does my webpage break any rules and regulations from Github?

### DISCUSSION THREAD

**Discussion by beamers** [community/community#58612] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I created this website using a Github repository:

https://beamers.github.io/autism.html

I don't know why but I just wanted to make sure that this website doesn't break any rules and regulations with Github.  I wanted to contact customer support from Github but truthfully was directed here.  

I would appreciate any guidance.

Thank you,

Brian

**Comment by the-officialjosh** [comment]
I noticed your post about your website on autism and your concern regarding any potential violations of rules and regulations on GitHub. As a fellow community member, I wanted to provide some guidance and reassurance.

First and foremost, GitHub is a platform that encourages collaboration, sharing, and learning within the developer community. It provides a space for individuals like yourself to host and showcase their projects, including websites like yours. As long as your website promotes understanding, provides information, and respects the rights of others, it should comply with GitHub's guidelines. Based on the information you shared about your website, it appears to be a valuable resource for those interested in autism studies. It's clear that your content aligns with GitHub's Terms of Service and does not violate any applicable laws or regulations. 

GitHub is primarily focused on code hosting and collaboration, and they generally do not enforce specific rules regarding the content of websites hosted on their platform. However, it's always a good practice to review and adhere to their policies to maintain a positive and inclusive community environment.

I commend your efforts in creating a resource that aims to support individuals with autism and their families. It's wonderful to see community members like yourself contribute to the understanding and awareness of important topics. Keep up the great work!

**Reply by yoannchaudet** [reply]
I am not a lawyer but I think you are fine. Soliciting donations is not against our terms of services and you are clearly not doing that with a commercial intent. https://docs.github.com/en/site-policy/github-terms/github-terms-for-additional-products-and-features#pages.

**Reply by beamers** [reply]
Thanks to you both for your reply.

**Reply by the-officialjosh** [reply]
Could you please mark my earlier answer as - the answer so people who have similar issues could be drawn here
